### ISSUE TITLE
Add copy to clipboard feature

### ISSUE THREAD

**Issue by midlan** [meetfranz/franz#1365]
I wish Franz to include feature to copy image in clipboard. Now it is only possible to copy the image url, but in some cases it's useless because to see the image url it is required to login on some services.

Now it's pretty anoying first to download image, open it and copy it in clipboard. I don't see reason why Franz can't do it direct. As I know, it uses the chrome engine and chrome contains the feature.

**Comment by lolmaus** [comment]
This feature I'm lacking badly.

Especially so, given that many webapps use blobs to display images, so "Open image in browser" does not work either.

I have to work around by saving the image on the file system, which is terribly inconvenient and leaves garbage (and sometimes compromising evidence) behind.

CC @adlk.

**Comment by CaptObvious** [comment]
I'm 99.9% convinced that this USED to be there and was removed.  I can't imagine why.

**Comment by Gaboris** [comment]
@CaptObvious
Yeah it was part of Electron's default context menu, but it was bulky and had some problems so it was replaced with a custom one, but Electron doesn't allow parts of the original menu to be included and making a "Copy image to clipboard" function seems to be harder than it seems so it had to be left out.

As far as I can remember it'll be added in once they can figure out how to make one by themselves.

**Comment by devcline** [comment]
THIS IS must have feature! Why have url if you can copy

**Comment by lolmaus** [comment]
> have url

Many messengers use blobs for images: internal links that are useless when copied. So "copy URL" is not an option either.

**Comment by devcline** [comment]
The hack solution is print screen the area and paste. But its not the best way

**Comment by devcline** [comment]
up!

**Comment by midlan** [comment]
still miss it

**Comment by qstyler** [comment]
Really miss this feature. Please implement.

**Comment by warshell** [comment]
up!

**Comment by lolmaus** [comment]
Just use Ferdi. This feature has been available in Ferdi for years, along with many other missing features and fixed bugs.
